{
    "type": "bundle",
    "id": "bundle--49795a9e-1859-42ae-b8cd-147912961829",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--1b966008-84ab-49f4-b09f-1a86da25fe7c",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2021-03-13T20:11:56.697594Z",
            "modified": "2021-03-13T20:11:56.697594Z",
            "name": "CVE-2019-13233",
            "description": "In arch/x86/lib/insn-eval.c in the Linux kernel before 5.1.9, there is a use-after-free for access to an LDT entry because of a race condition between modify_ldt() and a #BR exception for an MPX bounds violation.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2019-13233"
                }
            ],
            "object_marking_refs": [
                "marking-definition--62fd3f9b-15f3-4ebc-802c-91fce9536bcf"
            ]
        }
    ]
}